src/adagrams.js

```diff
@@ -1,15 +1,110 @@
+const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+};
+
+const SCORE_CHART = {
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1,
+    'T': 1, 'D': 2, 'G': 2, 'B': 3, 'C': 3, 'M': 3, 'P': 3, 'F': 4, 'H': 4,
+    'V': 4, 'W': 4, 'Y': 4, 'k': 5, 'J': 8, 'X': 8, 'Q': 10, 'Z': 10
+}
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+    // Implement this method for wave 1
+    const allLetters = []
+    const hand = []
+    for (let key of Object.keys(LETTER_POOL)) {
```

Comment:
Since `key` isn't reassigned in your loop, we should use const
```suggestion
    for (const key of Object.keys(LETTER_POOL)) {
```